app/views/restaurants/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<div style="margin:0 auto; width: 50%; background-color: grey; border: 2px solid black; text-align: center;" >
+<h1>Add Restaurant</h1>

```

Comment:
🔨 indentation is off, everything inside the `<div></div>` should be indented on level